client/state/selectors/is-nav-unification-new-user.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Internal dependencies
+ */
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+
+const NEW_USER_ID_THRESHOLD = 202731080; // ID of user who first registered on March 5, 2021.
```

Comment:
Is there any info in the FG about this? Would be good to have a canonical ref.

Reply:
Going to add a new section to the FAQ page.